serverlessworkflow/modules/ROOT/pages/cloud/quarkus/build-workflow-images-with-tekton.adoc

```diff
@@ -1 +1,143 @@
-//= Building Workflow Images with Tekton Pipelines
\ No newline at end of file
+= Building Workflows Images with Tekton
+:compat-mode!:
+// Metadata:
+:description: Building Workflow Images with Tekton
+:keywords: kogito, workflow, serverless, operator, kubernetes, minikube, openshift, containers, tekton, pipelines
+
+This document describes how to build a Workflow Application on a Kubernetes cluster using the Tekton pipeline.
+
+
+.Prerequisites
+You have a git repo containing your workflow and the dockerfile to build the workflow as a docker/podman image.
+
+[NOTE]
+====
+
+If you are using Openshift, you can install the Red Hat Pipeline Operator. If you are using Minikube, you can follow this link:https://github.com/tektoncd/pipeline/blob/main/docs/developers/local-setup.md#using-minikube[tekton guide for Minikube].
+If you are using Kind you can follow this link:https://github.com/tektoncd/plumbing/tree/main/hack[Tekton guide for Kind]. If you want to use it on a plain kubernetes cluster, you can follow this link:https://tekton.dev/docs/pipelines/install/[Tekton guide for kubernetes].
+====
+
+
+== Create the pipeline
+In the `kogito-serverless-operator-system` namespace, where the Kogito Serverless Operator is deployed in the pipelines menu, open the pipeline item, create it, and copy the following yaml file.
```

Comment:
Since into the requirements there isn't the Operator, I'll generalize suggesting to the user to select a namespace or creating a new one.
What do you think?

Reply:
ok